resources/Materials/TestSuite/nodedefs/convert_boolean_surfaceshader.mtlx

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <convert name="convert_boolean_surfaceshader" type="surfaceshader" version="1.0">
```

Comment:
Would these new tests make more sense as part of the existing `convert.mtlx` file in the test suite?

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/resources/Materials/TestSuite/stdlib/channel/convert.mtlx

Reply:
These were generated so I left them as is, but good suggestion to group them together. I'll make the change.